app/presenters/tree_builder_buttons.rb

```diff
@@ -35,18 +43,4 @@ def x_get_tree_custom_kids(object, count_only, _options)
     )
     count_only_or_objects(count_only, objects)
   end
-
-  def buttons_node_image(node)
-    case node
-    when 'ExtManagementSystem' then {:icon  => 'pficon pficon-server'}
-    when 'CloudTenant'         then {:icon  => 'pficon-cloud-tenant'}
-    when 'EmsCluster'          then {:icon  => 'pficon pficon-cluster'}
-    when 'Host'                then {:icon  => 'pficon pficon-screen'}
-    when 'MiqTemplate'         then {:icon  => 'ff ff-template'}
-    when 'Service'             then {:icon  => 'pficon pficon-service'}
-    when 'Storage'             then {:icon  => 'fa fa-database'}
-    when 'Vm'                  then {:icon  => 'pficon pficon-virtual-machine'}
-    when 'PhysicalServer'      then {:icon  => 'pficon pficon-enterprise'}
```

Comment:
Ok, ignoring `PhysicalServer` for now, since it's not visible there anyway.

@AndreyMenezes, @GabrielSVinha Once you do have something, please make sure the icon in the physical server decorator is the one that should be used. We will not support having a different icon in the list view and here.